app/models/deletion.rb

```diff
@@ -0,0 +1,38 @@
+class Deletion < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Username can change. There's not a `dependent` relationship on `has_many :deletions` so these records will stay in the DB.